### ISSUE TITLE
TypeError when OSBS extra directory is not provided

### ISSUE THREAD

**Issue by goldmann** [cekit/cekit#542]
**Describe the bug**
Stacktrace is thrown when a modules is beibg prepared. Probably just after this task.

**To reproduce**

Clone https://github.com/jboss-container-images/openjdk and run:

```
$ cekit --verbose build --dry-run osbs
```

**Expected behavior**
Build should not fail.

**Logs**
```
$ cekit --verbose build --dry-run osbs
2019-06-14 11:10:31,826 cekit        DEBUG    Running version 3.1.0
2019-06-14 11:10:31,826 cekit        DEBUG    Configuring CEKit...
2019-06-14 11:10:31,826 cekit        DEBUG    Removing dirty directory: 'target/image/modules'
2019-06-14 11:10:31,830 cekit        DEBUG    Removing dirty directory: 'target/repo'
2019-06-14 11:10:31,839 cekit        INFO     You are running on known platform: Fedora 30 (Thirty)
2019-06-14 11:10:31,839 cekit        DEBUG    Checking CEKit core dependencies...
2019-06-14 11:10:31,839 cekit        DEBUG    Checking if 'git' dependency is provided...
2019-06-14 11:10:31,839 cekit        DEBUG    CEKit dependency 'git' provided via the '/usr/bin/git' executable.
2019-06-14 11:10:31,839 cekit        DEBUG    All dependencies provided!
2019-06-14 11:10:31,917 cekit        INFO     Generating files for OSBS engine
2019-06-14 11:10:31,917 cekit        INFO     Initializing image descriptor...
2019-06-14 11:10:31,917 cekit        DEBUG    Checking CEKit generate dependencies...
2019-06-14 11:10:31,917 cekit        DEBUG    Checking if 'odcs-client' dependency is provided...
2019-06-14 11:10:31,917 cekit        DEBUG    Required CEKit library 'odcs-client' was found as a 'odcs' module!
2019-06-14 11:10:31,917 cekit        DEBUG    All dependencies provided!
2019-06-14 11:10:31,917 cekit        DEBUG    Removing old target directory
2019-06-14 11:10:31,919 cekit        DEBUG    Reading descriptor from 'image.yaml' file...
2019-06-14 11:10:31,929 cekit        DEBUG    Downloading module repository: 'cct_module'
2019-06-14 11:10:31,929 cekit        INFO     Preparing resource 'cct_module'
2019-06-14 11:10:31,929 cekit        DEBUG    Running 'git clone --depth 1 https://github.com/jboss-openshift/cct_module.git target/repo/cct_module.git-master -b master'
2019-06-14 11:10:33,826 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/os-java-run/module.yaml' file...
2019-06-14 11:10:33,829 cekit        DEBUG    Adding module 'os-java-run', path: 'target/repo/cct_module.git-master/os-java-run'
2019-06-14 11:10:33,829 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/os-logging/module.yaml' file...
2019-06-14 11:10:33,831 cekit        DEBUG    Adding module 'os-logging', path: 'target/repo/cct_module.git-master/os-logging'
2019-06-14 11:10:33,831 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/os-jws-secure-mgmt-console/module.yaml' file...
2019-06-14 11:10:33,833 cekit        DEBUG    Adding module 'os-jws-secure-mgmt-console', path: 'target/repo/cct_module.git-master/os-jws-secure-mgmt-console'
2019-06-14 11:10:33,833 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/os-java-s2i/module.yaml' file...
2019-06-14 11:10:33,835 cekit        DEBUG    Adding module 'os-java-s2i', path: 'target/repo/cct_module.git-master/os-java-s2i'
2019-06-14 11:10:33,836 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/jboss-maven/module.yaml' file...
2019-06-14 11:10:33,837 cekit        DEBUG    Adding module 'jboss-maven', path: 'target/repo/cct_module.git-master/jboss-maven'
2019-06-14 11:10:33,838 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/openshift-layer/module.yaml' file...
2019-06-14 11:10:33,840 cekit        DEBUG    Adding module 'openshift-layer', path: 'target/repo/cct_module.git-master/openshift-layer'
2019-06-14 11:10:33,840 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/os-jws-chmod/module.yaml' file...
2019-06-14 11:10:33,842 cekit        DEBUG    Adding module 'os-jws-chmod', path: 'target/repo/cct_module.git-master/os-jws-chmod'
2019-06-14 11:10:33,843 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/os-jws-logging/module.yaml' file...
2019-06-14 11:10:33,860 cekit        DEBUG    Adding module 'os-jws-logging', path: 'target/repo/cct_module.git-master/os-jws-logging'
2019-06-14 11:10:33,861 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/os-partition/module.yaml' file...
2019-06-14 11:10:33,863 cekit        DEBUG    Adding module 'os-partition', path: 'target/repo/cct_module.git-master/os-partition'
2019-06-14 11:10:33,863 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/products/kieserver6x/module.yaml' file...
2019-06-14 11:10:33,867 cekit        DEBUG    Adding module 'product.kieserver6x', path: 'target/repo/cct_module.git-master/products/kieserver6x'
2019-06-14 11:10:33,867 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/products/sso71/module.yaml' file...
2019-06-14 11:10:33,870 cekit        DEBUG    Adding module 'product.sso71', path: 'target/repo/cct_module.git-master/products/sso71'
2019-06-14 11:10:33,871 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/products/datavirt6x/module.yaml' file...
2019-06-14 11:10:33,874 cekit        DEBUG    Adding module 'product.datavirt6x', path: 'target/repo/cct_module.git-master/products/datavirt6x'
2019-06-14 11:10:33,875 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/os-sso/module.yaml' file...
2019-06-14 11:10:33,877 cekit        DEBUG    Adding module 'os-sso', path: 'target/repo/cct_module.git-master/os-sso'
2019-06-14 11:10:33,877 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/os-jws-launch/module.yaml' file...
2019-06-14 11:10:33,880 cekit        DEBUG    Adding module 'os-jws-launch', path: 'target/repo/cct_module.git-master/os-jws-launch'
2019-06-14 11:10:33,880 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/os-jws8-conffiles/module.yaml' file...
2019-06-14 11:10:33,882 cekit        DEBUG    Adding module 'os-jws8-conffiles', path: 'target/repo/cct_module.git-master/os-jws8-conffiles'
2019-06-14 11:10:33,882 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/os-java-jolokia/module.yaml' file...
2019-06-14 11:10:33,884 cekit        DEBUG    Adding module 'os-java-jolokia', path: 'target/repo/cct_module.git-master/os-java-jolokia'
2019-06-14 11:10:33,884 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/os-jws-jolokia/module.yaml' file...
2019-06-14 11:10:33,886 cekit        DEBUG    Adding module 'os-jws-jolokia', path: 'target/repo/cct_module.git-master/os-jws-jolokia'
2019-06-14 11:10:33,886 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/os-sso71/module.yaml' file...
2019-06-14 11:10:33,890 cekit        DEBUG    Adding module 'os-sso71', path: 'target/repo/cct_module.git-master/os-sso71'
2019-06-14 11:10:33,891 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/jolokia/module.yaml' file...
2019-06-14 11:10:33,893 cekit        DEBUG    Adding module 'jolokia', path: 'target/repo/cct_module.git-master/jolokia'
2019-06-14 11:10:33,894 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/os-sso72/module.yaml' file...
2019-06-14 11:10:33,898 cekit        DEBUG    Adding module 'os-sso72', path: 'target/repo/cct_module.git-master/os-sso72'
2019-06-14 11:10:33,899 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/openshift-passwd/module.yaml' file...
2019-06-14 11:10:33,902 cekit        DEBUG    Adding module 'openshift-passwd', path: 'target/repo/cct_module.git-master/openshift-passwd'
2019-06-14 11:10:33,902 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/os-jws-deployments/module.yaml' file...
2019-06-14 11:10:33,904 cekit        DEBUG    Adding module 'os-jws-deployments', path: 'target/repo/cct_module.git-master/os-jws-deployments'
2019-06-14 11:10:33,904 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/os-jws-rm-root/module.yaml' file...
2019-06-14 11:10:33,906 cekit        DEBUG    Adding module 'os-jws-rm-root', path: 'target/repo/cct_module.git-master/os-jws-rm-root'
2019-06-14 11:10:33,906 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/java-alternatives/module.yaml' file...
2019-06-14 11:10:33,908 cekit        DEBUG    Adding module 'java-alternatives', path: 'target/repo/cct_module.git-master/java-alternatives'
2019-06-14 11:10:33,908 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/os-java-misc/module.yaml' file...
2019-06-14 11:10:33,911 cekit        DEBUG    Adding module 'os-java-misc', path: 'target/repo/cct_module.git-master/os-java-misc'
2019-06-14 11:10:33,911 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/os-jws-s2i/module.yaml' file...
2019-06-14 11:10:33,913 cekit        DEBUG    Adding module 'os-jws-s2i', path: 'target/repo/cct_module.git-master/os-jws-s2i'
2019-06-14 11:10:33,913 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/os-jws-rm-defaults/module.yaml' file...
2019-06-14 11:10:33,915 cekit        DEBUG    Adding module 'os-jws-rm-defaults', path: 'target/repo/cct_module.git-master/os-jws-rm-defaults'
2019-06-14 11:10:33,915 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/os-java-hawkular/module.yaml' file...
2019-06-14 11:10:33,917 cekit        DEBUG    Adding module 'os-java-hawkular', path: 'target/repo/cct_module.git-master/os-java-hawkular'
2019-06-14 11:10:33,917 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/os-jws-https/module.yaml' file...
2019-06-14 11:10:33,919 cekit        DEBUG    Adding module 'os-jws-https', path: 'target/repo/cct_module.git-master/os-jws-https'
2019-06-14 11:10:33,919 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/jboss/container/hawkular/api/module.yaml' file...
2019-06-14 11:10:33,931 cekit        DEBUG    Adding module 'jboss.container.hawkular.api', path: 'target/repo/cct_module.git-master/jboss/container/hawkular/api'
2019-06-14 11:10:33,931 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/jboss/container/hawkular/bash/module.yaml' file...
2019-06-14 11:10:33,938 cekit        DEBUG    Adding module 'jboss.container.hawkular.bash', path: 'target/repo/cct_module.git-master/jboss/container/hawkular/bash'
2019-06-14 11:10:33,938 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/jboss/container/user/module.yaml' file...
2019-06-14 11:10:33,941 cekit        DEBUG    Adding module 'jboss.container.user', path: 'target/repo/cct_module.git-master/jboss/container/user'
2019-06-14 11:10:33,941 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/jboss/container/jolokia/api/module.yaml' file...
2019-06-14 11:10:33,950 cekit        DEBUG    Adding module 'jboss.container.jolokia.api', path: 'target/repo/cct_module.git-master/jboss/container/jolokia/api'
2019-06-14 11:10:33,950 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/jboss/container/jolokia/bash/module.yaml' file...
2019-06-14 11:10:33,961 cekit        DEBUG    Adding module 'jboss.container.jolokia.bash', path: 'target/repo/cct_module.git-master/jboss/container/jolokia/bash'
2019-06-14 11:10:33,962 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/jboss/container/util/logging/bash/module.yaml' file...
2019-06-14 11:10:33,965 cekit        DEBUG    Adding module 'jboss.container.util.logging.bash', path: 'target/repo/cct_module.git-master/jboss/container/util/logging/bash'
2019-06-14 11:10:33,966 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/jboss/container/prometheus/api/module.yaml' file...
2019-06-14 11:10:33,969 cekit        DEBUG    Adding module 'jboss.container.prometheus.api', path: 'target/repo/cct_module.git-master/jboss/container/prometheus/api'
2019-06-14 11:10:33,969 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/jboss/container/prometheus/bash/module.yaml' file...
2019-06-14 11:10:33,976 cekit        DEBUG    Adding module 'jboss.container.prometheus.bash', path: 'target/repo/cct_module.git-master/jboss/container/prometheus/bash'
2019-06-14 11:10:33,977 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/jboss/container/proxy/api/module.yaml' file...
2019-06-14 11:10:33,982 cekit        DEBUG    Adding module 'jboss.container.proxy.api', path: 'target/repo/cct_module.git-master/jboss/container/proxy/api'
2019-06-14 11:10:33,982 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/jboss/container/openjdk/jdk/8/module.yaml' file...
2019-06-14 11:10:33,987 cekit        DEBUG    Adding module 'jboss.container.openjdk.jdk', path: 'target/repo/cct_module.git-master/jboss/container/openjdk/jdk/8'
2019-06-14 11:10:33,987 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/jboss/container/openjdk/jdk/11/module.yaml' file...
2019-06-14 11:10:33,993 cekit        DEBUG    Adding module 'jboss.container.openjdk.jdk', path: 'target/repo/cct_module.git-master/jboss/container/openjdk/jdk/11'
2019-06-14 11:10:33,994 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/jboss/container/openjdk/jdk/10/module.yaml' file...
2019-06-14 11:10:33,999 cekit        DEBUG    Adding module 'jboss.container.openjdk.jdk', path: 'target/repo/cct_module.git-master/jboss/container/openjdk/jdk/10'
2019-06-14 11:10:33,999 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/jboss/container/openjdk/base/module.yaml' file...
2019-06-14 11:10:34,001 cekit        DEBUG    Adding module 'jboss.container.openjdk.base', path: 'target/repo/cct_module.git-master/jboss/container/openjdk/base'
2019-06-14 11:10:34,001 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/jboss/container/jws/s2i/bash/module.yaml' file...
2019-06-14 11:10:34,004 cekit        DEBUG    Adding module 'jboss.container.jws.s2i.bash', path: 'target/repo/cct_module.git-master/jboss/container/jws/s2i/bash'
2019-06-14 11:10:34,005 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/jboss/container/maven/35/35scl/module.yaml' file...
2019-06-14 11:10:34,008 cekit        DEBUG    Adding module 'jboss.container.maven.35.bash', path: 'target/repo/cct_module.git-master/jboss/container/maven/35/35scl'
2019-06-14 11:10:34,008 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/jboss/container/maven/35/default/module.yaml' file...
2019-06-14 11:10:34,011 cekit        DEBUG    Adding module 'jboss.container.maven.35.bash', path: 'target/repo/cct_module.git-master/jboss/container/maven/35/default'
2019-06-14 11:10:34,011 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/jboss/container/maven/s2i/api/module.yaml' file...
2019-06-14 11:10:34,015 cekit        DEBUG    Adding module 'jboss.container.maven.s2i.api', path: 'target/repo/cct_module.git-master/jboss/container/maven/s2i/api'
2019-06-14 11:10:34,015 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/jboss/container/maven/s2i/bash/module.yaml' file...
2019-06-14 11:10:34,033 cekit        DEBUG    Adding module 'jboss.container.maven.s2i.bash', path: 'target/repo/cct_module.git-master/jboss/container/maven/s2i/bash'
2019-06-14 11:10:34,033 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/jboss/container/maven/api/module.yaml' file...
2019-06-14 11:10:34,057 cekit        DEBUG    Adding module 'jboss.container.maven.api', path: 'target/repo/cct_module.git-master/jboss/container/maven/api'
2019-06-14 11:10:34,058 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/jboss/container/maven/default/bash/module.yaml' file...
2019-06-14 11:10:34,063 cekit        DEBUG    Adding module 'jboss.container.maven.default.bash', path: 'target/repo/cct_module.git-master/jboss/container/maven/default/bash'
2019-06-14 11:10:34,064 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/jboss/container/s2i/core/api/module.yaml' file...
2019-06-14 11:10:34,079 cekit        DEBUG    Adding module 'jboss.container.s2i.core.api', path: 'target/repo/cct_module.git-master/jboss/container/s2i/core/api'
2019-06-14 11:10:34,080 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/jboss/container/s2i/core/bash/module.yaml' file...
2019-06-14 11:10:34,083 cekit        DEBUG    Adding module 'jboss.container.s2i.core.bash', path: 'target/repo/cct_module.git-master/jboss/container/s2i/core/bash'
2019-06-14 11:10:34,083 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/jboss/container/java/proxy/bash/module.yaml' file...
2019-06-14 11:10:34,087 cekit        DEBUG    Adding module 'jboss.container.java.proxy.bash', path: 'target/repo/cct_module.git-master/jboss/container/java/proxy/bash'
2019-06-14 11:10:34,088 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/jboss/container/java/s2i/bash/module.yaml' file...
2019-06-14 11:10:34,093 cekit        DEBUG    Adding module 'jboss.container.java.s2i.bash', path: 'target/repo/cct_module.git-master/jboss/container/java/s2i/bash'
2019-06-14 11:10:34,094 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/jboss/container/java/run/bash/module.yaml' file...
2019-06-14 11:10:34,102 cekit        DEBUG    Adding module 'jboss.container.java.run.bash', path: 'target/repo/cct_module.git-master/jboss/container/java/run/bash'
2019-06-14 11:10:34,103 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/jboss/container/java/jvm/api/module.yaml' file...
2019-06-14 11:10:34,116 cekit        DEBUG    Adding module 'jboss.container.java.jvm.api', path: 'target/repo/cct_module.git-master/jboss/container/java/jvm/api'
2019-06-14 11:10:34,116 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/jboss/container/java/jvm/bash/module.yaml' file...
2019-06-14 11:10:34,120 cekit        DEBUG    Adding module 'jboss.container.java.jvm.bash', path: 'target/repo/cct_module.git-master/jboss/container/java/jvm/bash'
2019-06-14 11:10:34,120 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/keycloak-layer/module.yaml' file...
2019-06-14 11:10:34,122 cekit        DEBUG    Adding module 'keycloak-layer', path: 'target/repo/cct_module.git-master/keycloak-layer'
2019-06-14 11:10:34,123 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/os-jws-db-drivers/module.yaml' file...
2019-06-14 11:10:34,125 cekit        DEBUG    Adding module 'os-jws-db-drivers', path: 'target/repo/cct_module.git-master/os-jws-db-drivers'
2019-06-14 11:10:34,125 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/s2i-common/module.yaml' file...
2019-06-14 11:10:34,128 cekit        DEBUG    Adding module 's2i-common', path: 'target/repo/cct_module.git-master/s2i-common'
2019-06-14 11:10:34,128 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/os-jws7-conffiles/module.yaml' file...
2019-06-14 11:10:34,130 cekit        DEBUG    Adding module 'os-jws7-conffiles', path: 'target/repo/cct_module.git-master/os-jws7-conffiles'
2019-06-14 11:10:34,131 cekit        DEBUG    Reading descriptor from '/home/goldmann/git/public/container-images/openjdk/target/repo/cct_module.git-master/dynamic-resources/module.yaml' file...
2019-06-14 11:10:34,134 cekit        DEBUG    Adding module 'dynamic-resources', path: 'target/repo/cct_module.git-master/dynamic-resources'
2019-06-14 11:10:34,134 cekit        DEBUG    Preparing module 'jboss.container.openjdk.jdk' required by 'centos/openjdk-8-centos7'.
2019-06-14 11:10:34,135 cekit        DEBUG    Preparing module 'jboss.container.user' required by 'jboss.container.openjdk.jdk'.
2019-06-14 11:10:34,135 cekit        DEBUG    Preparing module 'jboss.container.java.s2i.bash' required by 'centos/openjdk-8-centos7'.
2019-06-14 11:10:34,135 cekit        DEBUG    Preparing module 'jboss.container.maven.s2i.bash' required by 'jboss.container.java.s2i.bash'.
2019-06-14 11:10:34,136 cekit        DEBUG    Preparing module 'jboss.container.maven.s2i.api' required by 'jboss.container.maven.s2i.bash'.
2019-06-14 11:10:34,136 cekit        DEBUG    Preparing module 'jboss.container.s2i.core.api' required by 'jboss.container.maven.s2i.api'.
2019-06-14 11:10:34,136 cekit        DEBUG    Preparing module 'jboss.container.s2i.core.bash' required by 'jboss.container.maven.s2i.bash'.
2019-06-14 11:10:34,137 cekit        DEBUG    Preparing module 'jboss.container.s2i.core.api' required by 'jboss.container.s2i.core.bash'.
2019-06-14 11:10:34,137 cekit        DEBUG    Preparing module 'jboss.container.maven.default.bash' required by 'jboss.container.maven.s2i.bash'.
2019-06-14 11:10:34,137 cekit        DEBUG    Preparing module 'jboss.container.maven.api' required by 'jboss.container.maven.default.bash'.
2019-06-14 11:10:34,137 cekit        DEBUG    Preparing module 'jboss.container.java.jvm.bash' required by 'jboss.container.maven.default.bash'.
2019-06-14 11:10:34,138 cekit        DEBUG    Preparing module 'jboss.container.java.jvm.api' required by 'jboss.container.java.jvm.bash'.
2019-06-14 11:10:34,138 cekit        DEBUG    Preparing module 'jboss.container.java.proxy.bash' required by 'jboss.container.java.jvm.bash'.
2019-06-14 11:10:34,138 cekit        DEBUG    Preparing module 'jboss.container.proxy.api' required by 'jboss.container.java.proxy.bash'.
2019-06-14 11:10:34,138 cekit        DEBUG    Preparing module 'jboss.container.maven.35.bash' required by 'jboss.container.maven.default.bash'.
2019-06-14 11:10:34,138 cekit        WARNING  Module version not specified for jboss.container.maven.35.bash, using 3.5scl version.
2019-06-14 11:10:34,139 cekit        DEBUG    Preparing module 'jboss.container.util.logging.bash' required by 'jboss.container.maven.default.bash'.
2019-06-14 11:10:34,139 cekit        DEBUG    Preparing module 'jboss.container.util.logging.bash' required by 'jboss.container.maven.s2i.bash'.
2019-06-14 11:10:34,139 cekit        DEBUG    Preparing module 'jboss.container.java.run.bash' required by 'jboss.container.java.s2i.bash'.
2019-06-14 11:10:34,139 cekit        DEBUG    Preparing module 'jboss.container.java.jvm.bash' required by 'jboss.container.java.run.bash'.
2019-06-14 11:10:34,139 cekit        DEBUG    Preparing module 'jboss.container.util.logging.bash' required by 'jboss.container.java.run.bash'.
2019-06-14 11:10:34,139 cekit        DEBUG    Preparing module 'jboss.container.hawkular.bash' required by 'jboss.container.java.s2i.bash'.
2019-06-14 11:10:34,140 cekit        DEBUG    Preparing module 'jboss.container.hawkular.api' required by 'jboss.container.hawkular.bash'.
2019-06-14 11:10:34,140 cekit        DEBUG    Preparing module 'jboss.container.jolokia.bash' required by 'jboss.container.java.s2i.bash'.
2019-06-14 11:10:34,140 cekit        DEBUG    Preparing module 'jboss.container.jolokia.api' required by 'jboss.container.jolokia.bash'.
2019-06-14 11:10:34,140 cekit        DEBUG    Preparing module 'jboss.container.prometheus.bash' required by 'jboss.container.java.s2i.bash'.
2019-06-14 11:10:34,141 cekit        DEBUG    Preparing module 'jboss.container.prometheus.api' required by 'jboss.container.prometheus.bash'.
2019-06-14 11:10:34,141 cekit        DEBUG    Preparing module 'jboss.container.util.logging.bash' required by 'jboss.container.java.s2i.bash'.
Traceback (most recent call last):
  File "/usr/bin/cekit", line 11, in <module>
    load_entry_point('cekit==3.1.0', 'console_scripts', 'cekit')()
  File "/usr/lib/python3.7/site-packages/click/core.py", line 763, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python3.7/site-packages/click/core.py", line 716, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python3.7/site-packages/click/core.py", line 1136, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib/python3.7/site-packages/click/core.py", line 1136, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib/python3.7/site-packages/click/core.py", line 955, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib/python3.7/site-packages/click/core.py", line 554, in invoke
    return callback(*args, **kwargs)
  File "/usr/lib/python3.7/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/usr/lib/python3.7/site-packages/cekit/cli.py", line 171, in build_osbs
    run_build(ctx, 'osbs')
  File "/usr/lib/python3.7/site-packages/cekit/cli.py", line 282, in run_build
    run_command(ctx, builder_impl)
  File "/usr/lib/python3.7/site-packages/cekit/cli.py", line 251, in run_command
    Cekit(common_params).run(clazz, params)
  File "/usr/lib/python3.7/site-packages/cekit/cli.py", line 341, in run
    command.execute()
  File "/usr/lib/python3.7/site-packages/cekit/builder.py", line 53, in execute
    self.prepare()
  File "/usr/lib/python3.7/site-packages/cekit/builders/osbs.py", line 86, in prepare
    super(OSBSBuilder, self).prepare()
  File "/usr/lib/python3.7/site-packages/cekit/builder.py", line 89, in prepare
    self.generator.generate(self.build_engine)
  File "/usr/lib/python3.7/site-packages/cekit/generator/osbs.py", line 31, in generate
    os.path.join(os.path.dirname(self._descriptor_path), self.image.osbs.extra_dir),
  File "/usr/lib64/python3.7/posixpath.py", line 94, in join
    genericpath._check_arg_types('join', a, *p)
  File "/usr/lib64/python3.7/genericpath.py", line 149, in _check_arg_types
    (funcname, s.__class__.__name__)) from None
TypeError: join() argument must be str or bytes, not 'NoneType'
```

**Environment information:**
 - 3.1.0

**Additional context**
None.

**Comment by goldmann** [comment]
This is related to #394 implemented in #519.

**Comment by goldmann** [comment]
@jmtd FYI
